### DISCUSSION TITLE
Granular git access in an NX Workspace.

### DISCUSSION THREAD

**Discussion by MalikBagwala** [nrwl/nx#11889] [General]
I want to give separate git access to NX applications, but since the workspace is tightly coupled (due to the use of workspace.json, package.json etc) I am unable to figure out how.

apps
   app1 -> developer1
   app2 -> developer2

Is there any way to achieve a smooth workflow with the above requirements, I have tried git sub modules and git sub-tree (subtree does not work because an nx application cannot be run in isolation)

Any advice or suggestions would be highly appreciated. 

@FrozenPandaz @supaspoida

**Comment by luii** [comment]
There is the concept of Code Owners which you could use.

1*) https://nx.dev/more-concepts/why-monorepos#scaling-your-organization-with-nx
2*) https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
